optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -254,51 +254,6 @@ def run(self, command, ignore_status=False, quiet=True, timeout=60):
                                                hosts=[self.hostname])
             return return_dict[self.hostname]
 
-    @staticmethod
-    def _run(command, ignore_status=False, quiet=True, timeout=60):
-        result = process.CmdResult()
-        start_time = time.time()
-        end_time = time.time() + (timeout or 0)   # Support timeout=None
-        # Fabric sometimes returns NetworkError even when timeout not reached
-        fabric_result = None
-        fabric_exception = None
-        while True:
-            try:
-                fabric_result = fabric.operations.run(command=command,
-                                                      quiet=quiet,
-                                                      warn_only=True,
-                                                      timeout=timeout,
-                                                      pty=False)
-                break
-            except fabric.network.NetworkError as details:
-                fabric_exception = details
-                if 'not found in known_hosts' in details.message:
```

Comment:
That check was lost with the commit that moved the method to the module: https://github.com/avocado-framework/avocado/commit/c06aea0f985cb9eea92bc5fde135aae0be36aa2f
But we already gave the 'unknown hosts' a good treatment after that commit, so I think we're safe in that regard.